src/components/Card/Card.js

```diff
@@ -1,23 +1,57 @@
 import styled from 'styled-components/macro'
+import { useState } from 'react'
 import PropTypes from 'prop-types'
 
 Card.propTypes = {
   image: PropTypes.node.isRequired,
   title: PropTypes.string,
   text: PropTypes.string,
+  adress: PropTypes.string,
+  openingHours: PropTypes.string,
+  entry: PropTypes.string,
 }
 
-export default function Card({ text, title, image }) {
+export default function Card({
+  text,
+  title,
+  image,
+  adress,
+  openingHours,
+  entry,
+}) {
+  const [isExpanded, setIsExpanded] = useState(false)
+
   return (
-    <WrapperList>
-      <img src={image} alt="" />
-      <h2>{title}</h2>
-      <p>{text}</p>
-    </WrapperList>
+    <WrapperCardList isExpanded={isExpanded}>
+      <img isExpanded={isExpanded} src={image} alt="" />
+      <h2 isExpanded={isExpanded}>{title}</h2>
+      <p isExpanded={isExpanded}>{text}</p>
```

Comment:
Why do you add `isExpanded` to these elements? It's not needed and you don't do anything with it.